packages/clay-management-toolbar/src/ActiveState.tsx

```diff
@@ -0,0 +1,248 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayButton from '@clayui/button';
+import ClayForm from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import ClayLink from '@clayui/link';
+import Item from './Item';
+import ItemList from './ItemList';
+import React from 'react';
+import {ClayDropDownWithBasicItems} from '@clayui/drop-down';
+import {IItem} from '@clayui/drop-down/src/DropDownWithBasicItems';
```

Comment:
Can we strip out `/src/DropDownWithBasicItems` here?

Reply:
Yeah, I think we can remove it now since we won't have `DropDownWithBasicItems` anymore.